lib/active_admin/async_exporter/services/aws_s3_service.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module ActiveAdmin
+  module AsyncExporter
+    module Services
+      class AwsS3Service
+        def initialize(file)
+          @file = file
+          @s3 = Aws::S3::Resource.new
```

Comment:
How are we handling the API keys?

Reply:
The AWS API key? It needs to be defined in the AWS initializer. I could add it in the README or add a new generator in order to be able to create that initializer in case the Dev doesn't have it yet